src/services/ebic/index.jsx

```
@@ -0,0 +1,306 @@
+import { useEffect, useRef, useState } from "react";
```

ESLint

[error] 'useRef' is defined but never used.

(no-unused-vars)